src/main/java/de/retest/recheck/cli/utils/ErrorHandler.java

```diff
@@ -26,16 +27,24 @@ public static void handle( final Exception e ) {
 			log.debug( "Stack trace:", e );
 			return;
 		}
-		if ( e instanceof IOException ) {
-			log.error( "An error occurred while loading the test report.", e );
-			return;
-		}
 		if ( e instanceof KryoException ) {
 			log.error( "The report was created with another, incompatible recheck version.\n"
 					+ "Please use the same recheck version to load a report with which it was generated." );
 			log.debug( "Stack trace:", e );
 			return;
 		}
+		if ( e instanceof UncheckedIOException ) {
+			log.error( "\n{}", e.getMessage() );
+			return;
+		}
+		if ( e instanceof NullPointerException ) {
```

Comment:
In which case does a NPE occur? Can this be just rethrown as a RuntimeException as this seems to be a valid exception within the code and nothing that can happen because of users' fault.